index.html

```diff
@@ -259,6 +259,28 @@ <h6 class="description">Hello I’m a software developer! I can help you build a
       </section>
     </section>
   </div>
-       </section>
+</section>
+<footer class="form-cont">
+  <form action="https://formspree.io/f/mrgdkyla" method="post">
+    <h2 class="h2">I'm always<br>interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+    <input type="text" id="name" placeholder="Full name" required maxlength="30" name="full name"><div class="line"></div>
+    <input type="email" placeholder="Email address" required><div class="line" name="email"></div>
+    <textarea name="text" maxlength="500" required placeholder="Enter text here"></textarea>
+    <button type="submit" class="get-in see1">Get in touch</button>
+  </form>
+  <div class="di"></div>
+  <section class="social">
+    <ul class="ico-ul">
+    <li class="git ico">
+      <img src="./images/icon2 twi.png" alt="">
+    </li>
+      <li class="lin ico"><img src="./images/icon2 li.png"></li>
+      <li class="ter ico"><img src="./images/angelist 1.png" alt=""></li>
+      <li class="twitter ico"><img src="./images/icontwi2.png" alt=""></li>
+      <li class="mess ico"><img src="./images/Vector.png" alt=""></li>
```

Comment:
- [ ] Please as required by the previous reviewer each social link should be in a `<a>` tag for best semantics tags because all icons links should be used to access their specific social links. Kindly fix it 🙃